rlclientlib/schema/v2/Event.fbs

```diff
@@ -10,6 +10,7 @@ table Event {
 
 table BatchMetadata {
     content_encoding: string; //valid values: IDENTITY and DEDUP
+	original_event_count: uint64;
```

Comment:
uint32 should be enough. Right?

Reply:
As per this conversation https://github.com/VowpalWabbit/reinforcement_learning/pull/448#discussion_r807543777, uint64_t seems more reasonable